scripts/install_docker.sh

```diff
@@ -3,3 +3,5 @@ set -ex
 
 curl -sSL https://get.docker.com/ubuntu/ | sh
 sudo gpasswd -a vagrant docker
+docker build -t android-base /vagrant/docker/android-base
+docker build -t android-studio /vagrant/docker/android-studio
```

Comment:
Made that change in my last commit actually.  They are not called from anywhere but I use them to build and run the images.  Also they are referenced in `README` for how to build and run the images yourself.